config/scheduler/config.yaml

```diff
@@ -24,6 +24,10 @@ schedulerMetrics:
     - "memory"
     - "ephemeral-storage"
     - "nvidia.com/gpu"
+  resourceRenaming:
+    nvidia.com/gpu: "gpu"
+    amd.com/gpu: "gpu"
+    ephemeral-storage: "ephemeral_storage"
```

Comment:
Is `ephemeralStorage` more idiomatic? I don't know what people normally do in Prometheus.